<ui version="4.0" >
 <class>DialogAdres</class>
 <widget class="QDialog" name="DialogAdres" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Adres Bilgileri</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="ikon.qrc" >:/ikonlar/contents.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Daimi Adres:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Geçici Adres:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTextEdit" name="TeDaimiAdres" />
   </item>
   <item row="1" column="1" >
    <widget class="QTextEdit" name="TeGeciciAdres" />
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="PbOnayla" >
     <property name="text" >
      <string>Onayla</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="Pbiptal" >
     <property name="text" >
      <string>İptal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ikon.qrc" />
 </resources>
 <connections/>
</ui>
